src/apps/common/ns_turn_utils.c

```diff
@@ -46,6 +46,15 @@
 
 #include <signal.h>
 
+#include <unistd.h>
+#include <sys/syscall.h>
+
+#ifndef SYS_gettid
+#error "SYS_gettid unavailable on this system"
```

Comment:
We do not want compilation to break here
Can we have default implementation?

Reply:
@eakraly Sorry it took a few days, been a busy week.  I've modified the PR so that if SYS_gettid doesn't exist then it will still compile, and logs will be same as before this commit (ie: no TID in logs).  Thanks for all your efforts in getting my PR's merged.  :)